lib/xMsExampleExtractor.js

```diff
@@ -149,7 +149,7 @@ class xMsExampleExtractor {
                   let infoFromOperation = paths[path][requestMethodFromRecording.toLowerCase()];
                   if (typeof infoFromOperation != 'undefined') {
                     //need to consider each method in operation
-                    let fileName = recordingFileName.split('/');
+                    let fileName = recordingFileName.split(/[/\\]/);
```

Comment:
Sure. Will check that. But I wanted to allow both kind of inputs "windows-style" and "Linux-style" to be parsed successfully. Will check if that happens using above option.